types/extra-types.d.ts

```diff
@@ -13,7 +13,7 @@ interface PolymerElementPropertiesMeta {
   computed?: string;
   reflectToAttribute?: boolean;
   notify?: boolean;
-  observer?: string;
+  observer?: string|((val: any, old: any) => void);
```

Comment:
are we ok doing this here? for polymer/polymer-decorators#23. though the externs js has `function(*, *)` or some such thing.

Reply:
Yes, I think this is correct.